charts/epinio/100.0.0+up0.9.1/templates/default-app-namespace.yaml

```diff
@@ -0,0 +1,22 @@
+---
+# Default namespace
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: workspace
+  annotations:
+    linkerd.io/inject: enabled
+  labels:
+    # Instruct kubed to copy image pull secrets over.
+    kubed-sync: "registry-creds"
+    app.kubernetes.io/component: "epinio-namespace"
```

Comment:
Does this need to be managed by Helm? Helm has two issues with namespace resource management:
1. When the chart is uninstalled, so is the namespace. Therefore uninstalling and re-installing the namespace will nuke all user data in that namespace
2. Namespaces created in this way won't be assigned a Project by default in Rancher (or might drop their project on an upgrade if it was added after deployment)

If we need this namespace to be created, we should create the namespace only if it does not exist, like how we handle the Alertmanager secret in Monitoring: https://github.com/rancher/charts/blob/389a06f64c63a8149e70e871d5f965c2d079dca0/charts/rancher-monitoring/100.1.3%2Bup19.0.3/templates/alertmanager/secret.yaml#L1-L4

Reply:
Will talk with the team.